widget/embedded/src/containers/Wallets/Wallets.tsx

```diff
@@ -40,6 +41,10 @@ function Main(props: PropsWithChildren<PropTypes>) {
   const onConnectWalletHandler = useRef<OnConnectHandler>();
   useSyncStoresWithConfig();
 
+  useEffect(() => {
+    void fetchMeta().catch();
```

Comment:
```suggestion
    void fetchMeta().catch(console/log);
```
I know you just moved the code, but completely ignoring the error will make frustrations in future. we should see the error in console at least.

Reply:
fixed.